packages/notebook-extension/src/index.ts

```diff
@@ -1123,6 +1125,72 @@ const activeCellTool: JupyterFrontEndPlugin<void> = {
   }
 };
 
+/**
+ * A plugin to add "Notebook (no kernel)" to the "Open With" context menu.
+ *
+ * Follows a similar logic as the filebrowser plugin populating the "Open With" menu
+ * based on widget factories.
+ *
+ * TODO: investigate generalizing opening without a kernel to other file types that may be
+ * backed by a kernel.
+ */
+const openWithNoKernelPlugin: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/notebook-extension:open-with-no-kernel',
+  description: 'Adds the "Notebook (no kernel)" option to the Open With menu.',
+  requires: [IFileBrowserFactory],
+  optional: [ITranslator],
+  autoStart: true,
+  activate: (
+    app: JupyterFrontEnd,
+    factory: IFileBrowserFactory,
+    translator: ITranslator | null
+  ): void => {
+    const { tracker } = factory;
+    const trans = (translator ?? nullTranslator).load('jupyterlab');
+
+    let items: IDisposableMenuItem[] = [];
```

Comment:
I think it is not reassigned?

```suggestion
    const items: IDisposableMenuItem[] = [];
```

Reply:
Indeed, thanks!